Misc/NEWS.d/next/Library/2021-01-18-10-41-44.bpo-42944.RrONvy.rst

```diff
@@ -0,0 +1 @@
+Fix `random.Random.sample` when `counts` argument is not `None`.
```

Comment:
Travis CI/Azure pipelines checks are failing because you need double backticks here ;).
```suggestion
Fix ``random.Random.sample`` when ``counts`` argument is not ``None``.
```

Reply:
OK ! Thank you.